### PR TITLE
Created style-tweaks how-to

### PR DESCRIPTION
**Pull request by shoshseiden**
I created a how-to file for style-tweaks. Issue 101.

### CODE REVIEW

File: docs/how-tos/style-tweaks.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Style Tweaks
+
+
+## How to edit LESS and rebuild CSS
+
+To edit LESS you will need to go to the less file you wish to change. The LESS
+files are located in `static/src/.less/`. In this example, we will use
```
Comment (paltman) [inline docs/how-tos/style-tweaks.md]: It's just `static/src/less/`

File: docs/how-tos/style-tweaks.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+```
+npm install
+```
+
+To rebuild the site, you will run the following command:
+```
+npm run build && ./manage.py runserver
+```
+The build command will rebuild the CSS and the runserver command allow you to
+view your changes locally.
```
Comment (paltman) [inline docs/how-tos/style-tweaks.md]: I think it's worth mentioning running `npm run watch` in a separate terminal window than `./manage.py runserver` so changes will be rebuilt on save and you can just refresh your browser as you are developing styles.  But then to make a production build, you run `npm run build` before committing.

**Comment by paltman** [comment]
Looking great @shoshseiden just a few items of feedback and then we can merge!  thanks!

**Comment by shoshseiden** [comment]
Hey @paltman I just made the two corrections you provided in the feedback. Feel free to let me know if there is anything else you need me to do. Thanks!

**Comment by ossanna16** [comment]
Thanks @shoshseiden, you're awesome :) @paltman Do you think this can be merged now?

**Comment by paltman** [comment]
Thanks @shoshseiden

**Pull request merged**